src/Nethermind/Nethermind.State/VerkleStateTree.cs

```diff
@@ -72,6 +73,28 @@ public void SetCode(Address address, byte[] code)
         }
     }
 
+    public byte[] GetCode(Address address, Hash256? stateRoot = null)
```

Comment:
Made this `public` so that I could move the migration logic to a separate project (`Nethermind.VerkleMigration.Cli`)